### DISCUSSION TITLE
Header Icon tooltip

### DISCUSSION THREAD

**Discussion by hayata-yamamoto** [okaryo/TabTabTab#381] [Ideas]
Hi, my friend. 

Could tabtabtab display header menu tooltip? I'm afraid, This organization button sometimes confuse me due to not inspiring me what this feature is. 
https://github.com/okaryo/TabTabTab/blob/main/src/view/features/popup/components/Header.tsx#L152

If tooltips are displayed, users might easily start to use deeper and useful detailed features, like auto grouping and cleaning up. 
https://mui.com/material-ui/react-tooltip/

**Comment by okaryo** [comment]
Hi,

I’ve actually tried tooltips before, but in the narrow popup space, they felt a bit overwhelming. Also, once you navigate to the next page, it becomes clearer what each icon represents, so I decided not to use them.

I agree that the icon isn’t very intuitive, and "organization" doesn’t feel like the best name. I chose it to represent grouping and auto-deleting tabs, thinking of it as a way to tidy up. But I’m open to changing it in the future! If you have any better name ideas, feel free to share them.

Thanks for your feedback!
